index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style/style.css" />
+    <script
+      src="https://kit.fontawesome.com/14f4680366.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Artex</title>
+  </head>
+  <body>
+    <header>
+      <div class="icon-holder">
+        <img src="./images/Icons/fb.png" alt="facebook logo" />
+        <img src="./images/Icons/twit.png" alt="twitter logo" />
+        <h4>English</h4>
+        <h4>My Page</h4>
+        <h4>Logout</h4>
+      </div>
+      <nav class="mobile-nav">
+        <img
+          id="mobile-menu"
+          src="./images/Icons/Unionburger_icon.png"
+          alt="mobile menu"
+        />
+        <img
+          id="burger-x"
+          src="./images/Icons/burger-X.png"
+          alt="mobile menu"
+        />
+      </nav>
+      <nav class="desktop-nav">
+        <a href="#">
+          <img src="./images/Icons/logo.png" alt="Artex logo" />
+        </a>
+        <ul>
+          <li>
+            <a id="aboutlink" href="./about.html"><span>About</span></a>
+          </li>
+          <li><a href="">Program</a></li>
+          <li><a href="">Join</a></li>
+          <li><a href=""> Sponsor</a></li>
+          <li><a href=""> News</a></li>
+          <li id="campaign">Artex Campaign</li>
+        </ul>
+      </nav>
+    </header>
+    <section class="home-section">
+      <h4>"Hello! Aesthetes"</h4>
+      <div class="h1-div">
+        <h1>Artex GLOBAL ART EXHIBITION 2022</h1>
+      </div>
+      <p class="home-p1">
+        A joyful celebration believing in the value of openness and sharing,
+        creating positive change with people from over 80 countries is taking
+        place in December, in Greece
+      </p>
+      <h2>2022.12.23(FRI) ~ 24(SAT)</h2>
+      <p class="home-p2">@ Jeff Koons: Apollo Hydra, Greece</p>
+    </section>
```

Comment:
Good job working on the project requirement and neat U.I design 😎 but unfortunately there are still some issues you need to correct to get an approval. I noticed the date and event location on you home page are hidden and this is due to the background image you added in your main program section. Please correct this.

This is how your design looks:
![Captone 1](https://user-images.githubusercontent.com/79506105/196752282-644e71ef-2e52-4e42-a8d8-a5e4f5cbdbd9.png)

Please stick to this design: 
![Screen Shot 2022-10-19 at 18 20 51](https://user-images.githubusercontent.com/79506105/196752456-6fe9b009-9b30-468d-98e2-60f67e885ffc.png)